src/components/Catalog.vue

```diff
@@ -207,7 +213,7 @@ export default {
       required: true
     }
   },
-  components: { AssetTab, MetadataSidebar },
+  components: { AssetTab, ZarrMetadataTab, MetadataSidebar },
```

Comment:
Load component on demand only:

```suggestion
  components: {
    AssetTab,
    ZarrMetadataTab: () => import('./ZarrMetadataTab.vue'),
    MetadataSidebar
  },
```

Reply:
This change works as intended, but seems to create a parsing error with ESLint that prevents it from linting the rest of the file - is this something we should work out before pushing this change?

A seemingly similar issue being dealt with here:

https://stackoverflow.com/questions/47815775/dynamic-imports-for-code-splitting-cause-eslint-parsing-error-import